polaris-react/src/components/Frame/Frame.tsx

```diff
@@ -91,13 +98,21 @@ class FrameInner extends PureComponent<CombinedProps, State> {
     }
     this.setGlobalRibbonRootProperty();
     this.setOffset();
+    document.documentElement.classList.toggle(
```

Comment:
You're right, we don't want this any more. I was separating this PR up into the one from this morning that adds the global class. I'm going to close this PR and follow up with another that just adds the colors, because the commits are causing issues with CLA here